docker-compose.yml

```diff
@@ -39,7 +40,7 @@ services:
       - ./app-frontend/package.json:/opt/raster-foundry/app-frontend/package.json
       - ./app-frontend/src:/opt/raster-foundry/app-frontend/src
       - ./app-frontend/webpack.config.js:/opt/raster-foundry/app-frontend/webpack.config.js
-    command: npm run build-watch
+    command: npm run start
```

Comment:
I made the change because because package.json doesn't contain a `build-watch` script, causing the `app-frontend` container to exit:
```diff
$ diff --git a/docker-compose.yml b/docker-compose.yml
index 2cac9a3..cdaf089 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -40,7 +40,7 @@ services:
       - ./app-frontend/package.json:/opt/raster-foundry/app-frontend/package.json
       - ./app-frontend/src:/opt/raster-foundry/app-frontend/src
       - ./app-frontend/webpack.config.js:/opt/raster-foundry/app-frontend/webpack.config.js
-    command: npm run start
+    command: npm run build-watch
```

```bash
$ docker-compose up app-frontend
Creating rasterfoundry_app-frontend_1
Attaching to rasterfoundry_app-frontend_1
app-frontend_1    | npm info it worked if it ends with ok
app-frontend_1    | npm info using npm@3.10.8
app-frontend_1    | npm info using node@v6.8.1
app-frontend_1    | npm ERR! Linux 3.13.0-101-generic
app-frontend_1    | npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build-watch"
app-frontend_1    | npm ERR! node v6.8.1
app-frontend_1    | npm ERR! npm  v3.10.8
app-frontend_1    | 
app-frontend_1    | npm ERR! missing script: build-watch
app-frontend_1    | npm ERR! 
app-frontend_1    | npm ERR! If you need help, you may report this error at:
app-frontend_1    | npm ERR!     <https://github.com/npm/npm/issues>
app-frontend_1    | 
app-frontend_1    | npm ERR! Please include the following file with any support request:
app-frontend_1    | npm ERR!     /opt/raster-foundry/app-frontend/npm-debug.log
rasterfoundry_app-frontend_1 exited with code 1
```
I looked at the [blame](https://github.com/azavea/raster-foundry/blame/develop/app-frontend/package.json) for package.json, and it doesn't look like `build-watch` was ever included. I'm not sure how `npm run build-watch` ever worked.